src/resolvers-cassandra/Edges.js

```diff
@@ -10,12 +10,26 @@ module.exports = {
   },
 
   saveLocations(args, res){ // eslint-disable-line no-unused-vars
+    return new Promise((resolve, reject) => { // eslint-disable-line no-unused-vars
+      reject(
+        'This API call is no longer supported. ' +
+        'We now automatically filter events down to only those ' +
+        'locations defined in the geo-fence for your site'
+      );
+    });
```

Comment:
Yes, the idea is to have the cassandra-based resolvers to be 100% backwards compatible with the Fortis-v1 UI. This has the advantage that once all of our Fortis-v2 backend work is done, we can just hook up the v2 backend to the v1 frontend for a first milestone and then focus on revamping the frontend as a second milestone without having to worry about an in-flux backend.

The methods in /resolvers don't have to be changed as those are the Fortis-v1 implementations.